index.spec.js

```diff
@@ -11,24 +11,40 @@ describe('Testing function minVal()', () => {
         expect(minVal([1,2,34,0,-12])).not.toBeInstanceOf(Object);
     });
     it('Shoul get a minimal value from an array', () => {
-        expect(minVal([])).toBe(-12);
+        expect(minVal([1,2,3,4])).toBe(1);
+        expect(minVal([3,3,3,3])).toBe(3);
+        expect(()=>{minVal([])}).toThrow('Array cannot be empty');
     });
 });
 
 describe('Testing function timesTwo()', () => {
     it('returns an array where numbers are doubled', () => {
-        expect(timesTwo([1,2])).toBe([5,4]);
+        expect(Array.isArray(timesTwo([1,2]))).toBe(true);
+        expect(timesTwo([1,2])).toEqual([2,4]);
     });
 });
 
 describe('Testing function fizzBuzz()', () => {
     it('returns an array of the same length as a parameter', () => {
-        expect(fizzBuzz(10)).toHaveLength(7);
+        expect(fizzBuzz(10)).toHaveLength(10);
     });
     it('returns an array with some fizz, buzz and fizzbuzz', () => {
         expect(fizzBuzz(3)).toContain('fizz');
-        expect(fizzBuzz(3)).toContain('fizzbuzz');
+        expect(fizzBuzz(15)).toContain('fizzbuzz');
     })
 });
 
 
+describe('Testing function returnObjects()',()=>{
```

Comment:
Not sure I am happy with that many starter tests. It's a workshop, not an actual code 😄